_resources/assets/redirects

```diff
@@ -41,4 +41,5 @@
 /handbook/product/personas /handbook/marketing/personas 308
 /handbook/values /company/values 308
 
-
+/handbook/ce/support /handbook/support 308
+/handbook/ce/support/#slas /handbook/support#our-service-level-agreements-slas 308
```

Comment:
```suggestion

```
This rule isn't needed/won't work due to the fragment (# symbol and stuff behind it) never reaching the backend, only the client (browser) knows about it.

Reply:
You could add an additional anchor to the SLAs section on the support page to catch the old fragment.

Reply:
Ah got it. Will update that page, then, too. Thank you!